### ISSUE TITLE
Notify the user when Imagick is missing

### ISSUE THREAD

**Issue by curtchan** [Lorti/dominant-colors-lazy-loading-wordpress-plugin#24]
If server has imagick disabled, on calculation settings page there is "Do you want to calculate now?" even though there is no button or anything. Would be nice to have there direct disclaimer why it is like that.

**Comment by Lorti** [comment]
Yes, that’s a good idea. The README mentions that you have to have ImageMagick enabled, but that might easily be overlooked. Please feel free to open up a pull request.
